project/plugins.sbt

```diff
@@ -9,3 +9,4 @@ addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.6.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 addSbtPlugin("org.tpolecat" % "tut-plugin" % "0.6.10")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.15")
+addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.1.2")
```

Comment:
out of curiosity: why was this added?

Reply:
I accidentally checked it in. I use `bloop` while developing. So, I added it and forgot to remove it.

Reply:
@fthomas Is it Ok if I leave this plugin in there ?